clickhouse_driver/client.py

```diff
@@ -348,7 +348,7 @@ def execute_with_progress(
             )
 
     def execute_iter(
-            self, query, params=None, with_column_types=False,
+            self, query, chunk_size=1, params=None, with_column_types=False,
```

Comment:
It's better to add new keyword arguments to the end of existing

Reply:
Okay, I will put it to the end of existing.